src/modules/addMALscore.js

```diff
@@ -114,20 +114,14 @@ async function addMALscore(type,id){
 					adder()
 				}
 			}
-			if(window.GM_xmlhttpRequest){
+			if(GM_xmlhttpRequest){
```

Comment:
This doesn't work. Throws an error if no such thing is defined.

Reply:
oops, fixed it